src/jobs/generation/Utilities.groovy

```diff
@@ -616,6 +616,20 @@ class Utilities {
     }
 
     /**
+     * Adds a VSTS PR trigger
+     */
+    def static addVSTSPRTrigger(def job) {
+        job.with {
+            triggers {
+                TeamPRPushTrigger()
```

Comment:
Is the casing correct here?  Why is it camelcase for teamPushTrigger but TeamPRPushTrigger is different?